### ISSUE TITLE
Breaks on Pop!_OS 21.04

### ISSUE THREAD

**Issue by thebigG** [thebigG/Tasker#33]
Tested Tasker on a clean ISO image and got the following error when running the AppImage:
```
root@pop-os:/home/pop-os# ./Tasker-0.1.5-x86_64_8d39aa060f5c15e0aef66e40346aa958.AppImage 
/tmp/.mount_TaskerkE467L/usr/bin/Tasker: error while loading shared libraries: libQt5Multimedia.so.5: cannot open shared object file: No such file or directory

```

**Comment by thebigG** [comment]
This might be an issue with the way I'm building the AppImage. The same thing happens in Pop_os 20.04

**Comment by thebigG** [comment]
```
/Tasker-0.1.5-x86_64_8d39aa060f5c15e0aef66e40346aa958.AppImage 
/tmp/.mount_TaskerG5gf0A/usr/bin/Tasker: error while loading shared libraries: libQt5Charts.so.5: cannot open shared object file: No such file or directory

```

**Comment by thebigG** [comment]
This is mostly fixed by 5454073d62376fb00e49af64656153a4cacc746f. But the AudioHook does not work on Pop OS 21. Need to investigate this further.

**Comment by thebigG** [comment]
Everything seems to be working fine in Ubuntu 21. Seems as if I was using the wrong build, which is very possible.
